apps/src/assetManagement/assetPrefix.js

```diff
@@ -13,6 +13,8 @@ const ABSOLUTE_CDO_CURRICULUM_REGEXP = new RegExp(
   'i'
 );
 
+export const DATA_URL_PREFIX_REGEX = new RegExp('^data:image');
```

Comment:
super nit: we include the `/` in our other regex tests below (e.g., `'icon://'` on line 18) -- should we do the same here for consistency?

Reply:
hmmmm, nevermind, it looks like the forward slash _may_ be part of the mime type (like `data:image/gif`), but isn't always present. so don't add that forward slash!